addons/edi/models/edi_document.py

```diff
@@ -188,6 +190,22 @@ class EdiDocument(models.Model):
         copy=False,
         tracking=True,
     )
+    execution_state = fields.Selection(
```

Comment:
This should be an EDI document type config. Where is this used?

Reply:
Probably needs renaming.

Reply:
May be `processing_state`?
Figuring out the exact location to use this flag (may be in `autoprepare()`?)